### PR TITLE
"k8s-stern YML files included in  CI"

### PR DESCRIPTION
**Pull request by vibhor995**
Signed-off-by: vibhor995 <vvibhr995@gmail.com>

Created k8-stern yml files inside /roles which will download the stern binary file locally and run stern in the master node.  **(This is included in CI.yml also)**.

Please review:- @ksatchit @dargasudarshan

### CODE REVIEW

**Comment by vibhor995** [comment]
**output**

```$ ansible-playbook setup-stern.yml -vv
ansible-playbook 2.5.2
  config file = /home/giri/openebs/e2e/ansible/ansible.cfg
  configured module search path = [u'/home/giri/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /home/giri/.local/lib/python2.7/site-packages/ansible
  executable location = /home/giri/.local/bin/ansible-playbook
  python version = 2.7.12 (default, Dec  4 2017, 14:50:18) [GCC 5.4.0 20160609]
Using /home/giri/openebs/e2e/ansible/ansible.cfg as config file

PLAYBOOK: setup-stern.yml **********************************************************************************************************************
1 plays in setup-stern.yml

PLAY [localhost] *******************************************************************************************************************************

TASK [Gathering Facts] *************************************************************************************************************************
task path: /home/giri/openebs/e2e/ansible/setup-stern.yml:3
ok: [localhost]
META: ran handlers

TASK [k8s-stern : Get Current User] ************************************************************************************************************
task path: /home/giri/openebs/e2e/ansible/roles/k8s-stern/tasks/main.yml:3
changed: [localhost -> None] => {"changed": true, "cmd": ["whoami"], "delta": "0:00:00.027217", "end": "2018-05-31 09:11:27.027989", "rc": 0, "start": "2018-05-31 09:11:27.000772", "stderr": "", "stderr_lines": [], "stdout": "ubuntu", "stdout_lines": ["ubuntu"]}

TASK [k8s-stern : Get Log Aggregator] **********************************************************************************************************
task path: /home/giri/openebs/e2e/ansible/roles/k8s-stern/tasks/main.yml:9
ok: [localhost] => {"attempts": 1, "changed": false, "checksum_dest": "134ad605ffdf90f4969f84f33fb29f34ea4cc79f", "checksum_src": "134ad605ffdf90f4969f84f33fb29f34ea4cc79f", "dest": "/home/giri/openebs/e2e/ansible/roles/k8s-stern/files/stern", "gid": 1000, "group": "giri", "md5sum": "2252747b21615da02a587271dde55537", "mode": "0664", "msg": "OK (19976800 bytes)", "owner": "giri", "size": 19976800, "src": "/tmp/tmpJiHFLd", "state": "file", "status_code": 200, "uid": 1000, "url": "https://github.com/wercker/stern/releases/download/1.6.0/stern_linux_amd64"}

TASK [k8s-stern : Copy log aggregator to remote] ***********************************************************************************************
task path: /home/giri/openebs/e2e/ansible/roles/k8s-stern/tasks/main.yml:21
ok: [localhost -> None] => {"changed": false, "checksum": "134ad605ffdf90f4969f84f33fb29f34ea4cc79f", "dest": "/usr/local/bin/stern", "gid": 1000, "group": "ubuntu", "mode": "0744", "owner": "ubuntu", "path": "/usr/local/bin/stern", "size": 19976800, "state": "file", "uid": 1000}
META: ran handlers
META: ran handlers

PLAY RECAP *************************************************************************************************************************************
localhost                  : ok=4    changed=1    unreachable=0    failed=0

File: e2e/ansible/roles/k8s-stern/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  command: whoami
+  register: user
+  delegate_to: "{{groups['kubernetes-kubemasters'].0}}"
+
+
+- name: Get Log Aggregator
+  get_url:
+    url: "{{k8s_log_aggregator_url}}"
+    dest: "{{ playbook_dir }}/roles/k8s-stern/files/{{k8s_log_aggregator}}"
+    #dest: "{{k8s_log_aggregator}}"
```
Comment (dargasudarshan) [inline e2e/ansible/roles/k8s-stern/tasks/main.yml]: Remove commented lines, if you this it is not necessary.

**Comment by vibhor995** [comment]
@dargasudarshan  removed comment line.

**Comment by vibhor995** [comment]
$ stern
$ stern: command not found
``` output After running stern yml file 

ubuntu@kubemaster-01:~$ stern
Tail multiple pods and containers from Kubernetes

Usage:
  stern pod-query [flags]

Flags:
      --all-namespaces        If present, tail across all namespaces. A specific namespace is ignored even if specified with --namespace.
      --color string          Color output. Can be 'always', 'never', or 'auto' (default "auto")
      --completion string     Outputs stern command-line completion code for the specified shell. Can be 'bash' or 'zsh'
  -c, --container string      Container name when multiple containers in pod (default ".*")
      --context string        Kubernetes context to use. Default to current context configured in kubeconfig.
  -e, --exclude stringSlice   Regex of log lines to exclude
      --kubeconfig string     Path to kubeconfig file to use
  -n, --namespace string      Kubernetes namespace to use. Default to namespace configured in Kubernetes context
  -l, --selector string       Selector (label query) to filter on. If present, default to ".*" for the pod-query.
  -s, --since duration        Return logs newer than a relative duration like 5s, 2m, or 3h. Defaults to all logs.
      --tail int              The number of lines from the end of the logs to show. Defaults to -1, showing all logs. (default -1)
  -t, --timestamps            Print timestamps
  -v, --version               Print the version and exit

**Pull request merged**